nativelib/src/main/resources/addrinfo.c

```diff
@@ -0,0 +1,236 @@
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <netdb.h>
+#include <string.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <netinet/ip_icmp.h>
+#include <stdio.h>
+#include <stdbool.h>
+#include <stdlib.h>
+
+char* scalanative_host_to_ip(char* host) {
```

Comment:
As @muxanick provided windows version of the bindings I that's a better idea for sure. I will rewrite in Scala.